server/src/main/java/org/apache/druid/server/lookup/cache/LookupLoadingSpec.java

```diff
@@ -39,6 +43,10 @@
  */
 public class LookupLoadingSpec
 {
+
+  public static final String CTX_LOOKUP_LOADING_MODE = "lookupLoadingMode";
+  public static final String CTX_LOOKUPS_TO_LOAD = "lookupsToLoad";
```

Comment:
I think these constants should live in `Tasks` class instead.

Reply:
@kfaraz `Tasks` class isn't accessible from `server` module. We also need the constants accessible in `LookupLoadingSpec` class for example.
Any other alternate suggestions on where can these be moved?

Reply:
I see, we need these for the new method `createSpecFromContext()`. In that case, you can leave these constants here.